.github/workflows/linters.yml

```diff
@@ -18,14 +18,14 @@ jobs:
           sudo apt-get update -y
           sudo apt-get install -y wget gnupg2 git
           wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
-          sudo bash -c "echo 'deb https://apt.llvm.org/focal/ llvm-toolchain-focal-17 main' >> /etc/apt/sources.list"
+          sudo bash -c "echo 'deb https://apt.llvm.org/focal/ llvm-toolchain-focal-18 main' >> /etc/apt/sources.list"
           sudo apt-get update -y
-          sudo apt-get install -y clang-format-17
+          sudo apt-get install -y clang-format-18
       - name: Run clang-format
         run: ./bin/check-clang-format
         env:
-          CB_GIT_CLANG_FORMAT: /usr/bin/git-clang-format-17
-          CB_CLANG_FORMAT: /usr/bin/clang-format-17
```

Comment:
Homebrew updated clang to 18, so `git clang-format` is trying to enforce new rules and I gave up trying to find right flags to leave some bits untouched.